test/e2e-v2/cases/mysql/prometheus-mysql-exporter/otel-collector-config.yaml

```diff
@@ -22,19 +22,11 @@ receivers:
          static_configs:
            - targets: ['mysql-service:9104']
              labels:
-               host_name: showcase
+               host_name: root[root]
 processors:
   batch:
 
 exporters:
-  ####################################################################################
```

Comment:
These should be reverted? They are added recently

Reply:
I get it.